hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/test/java/org/apache/hadoop/yarn/server/federation/store/impl/FederationStateStoreBaseTest.java

```diff
@@ -879,7 +879,7 @@ public void testStoreNewMasterKey() throws Exception {
   }
 
   @Test
-  public void testGetMasterKeyByDelegationKey() throws YarnException, IOException {
+  public void testGetMasterKeyByDelegationKey() throws Exception {
```

Comment:
This is correct but do we need to add all this churn?

Reply:
The exception thrown by the method signature, I will recover. In the process of writing JunitTest code, I used `curatorFramework.getData().forPath` method to get data from ZK. This method throws an `Exception`, so I modified the exception throwing code of the method signature.

```
/**
     * Commit the currently building operation using the given path
     *
     * @param path the path
     * @return operation result if any
     * @throws Exception errors
     */
    public T forPath(String path) throws Exception;
```